tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java

```diff
@@ -937,4 +937,49 @@ void updateImageWithServerJRE(TestInfo testInfo) throws Exception {
             assertFalse(imageId.isEmpty(), "Image was not created: " + tagName);
         }
     }
+
+    /**
+     * Create an image with WDT Model only on OEL 8-slim
```

Comment:
Very minor point... They no longer call it "Oracle Enterprise Linux", it's just "Oracle Linux", so OL 8-slim.

Reply:
Thanks, fixed.